app/views/sufia/homepage/_home_header.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="home_call_action col-xs-12 col-sm-4 pull-right">
-  <% if can? :view_share_work, GenericWork %>
+  <% if Sufia.config.always_display_share_button || can_ever_create_works? %>
```

Comment:
I was wondering about consolidating the following into a single method:

```ruby```
Sufia.config.always_display_share_button || can_ever_create_works?
```

There are two layers of abstraction; A method call and a configuration option co-mingled in the view.